README.md

```diff
@@ -68,29 +68,34 @@ brew install k6
 
 ### Windows
 
-You can manually download and install the [official `.msi` installation package](https://dl.bintray.com/loadimpact/windows/k6-v0.31.1-amd64.msi) or, if you use the [chocolatey package manager](https://chocolatey.org/), follow [these instructions](https://bintray.com/repo/buildSettings?repoPath=%2Floadimpact%2Fchoco) to set up the k6 repository.
+If you use the [Chocolatey package manager](https://chocolatey.org/):
 
-### Linux
+```
+choco install k6
+```
 
-**Notice: Because [Bintray is being shutdown](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) we are going to start self-hosting our packages soon, before k6 v0.32.0. This means you will have to re-install them, since the old .rpm and .deb repos will stop working.**
+Otherwise you can manually download and install the [latest official `.msi` package](https://dl.k6.io/msi/k6-latest-amd64.msi).
 
-For Debian-based Linux distributions, you can install k6 from the private deb repo like this:
+### Linux
+
+For Debian-based Linux distributions like Ubuntu, you can install k6 from the private deb repo like this:
 
 ```bash
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
-echo "deb https://dl.bintray.com/loadimpact/deb stable main" | sudo tee -a /etc/apt/sources.list
+curl -s https://dl.k6.io/key.gpg | sudo apt-key add -
+echo "deb https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
```

Comment:
Should we include any information for users who already added the Bintray repo? I'll test if it's a smooth migration in that case, but we should probably mention it.

Reply:
Yes, we should probably mention something about the migration (and any manual steps that are required), both here and in the docs.

Reply:
Let me know if 90623b17 is clear enough. I'll add something similar to the docs.

From my tests there's no problem with having both the Bintray and our repos added at the same time, though installation will default to Bintray, I think because it was added first. Once it starts failing, it falls back to any alternative repos that carry the same package, so it's mostly a smooth process, for Debian/apt at least. On CentOS any `yum` operation fails if the Bintray repo fails, but once it's removed then it picks up the package from `dl.k6.io`.